lib/onebox/templates/preview_error_fragment_onebox.mustache

```diff
@@ -0,0 +1,4 @@
+<article class="onebox-warning-message">
+  {{{error_message}}}
+</article>
+<div style="clear: both"></div>
```

Comment:
I am surprised this works with our CSP rules. Would `<div class="clearfix"></div>` work? That should be in our scss.